frontend/templates/list-annotation-category.tmpl

```diff
@@ -0,0 +1,71 @@
+{{!-- Skalenwerte zuerst --}}
```

Comment:
The name of this partial seems weird to me. Shouldn't it rather be `list-annotation-content`? Of course this also applies to the variables bound to it.